### PR TITLE
Configure SonarCloud

### PR DESCRIPTION
**Pull request by Dabolus**
_Before you submit a pull request, please make sure you have read and understood the [contribution guidelines](https://github.com/TheFireCo/genkit-plugins/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/TheFireCo/genkit-plugins/blob/main/CODE_OF_CONDUCT.md)._

**This pull request is related to:**

- [ ] A bug
- [ ] A new feature
- [ ] Documentation
- [x] Other (please specify) code quality tracking

**I have checked the following:**

- [x] I have read and understood the [contribution guidelines](https://github.com/TheFireCo/genkit-plugins/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/TheFireCo/genkit-plugins/blob/main/CODE_OF_CONDUCT.md);
- [ ] I have added new tests (for bug fixes/features);
- [ ] I have added/updated the documentation (for bug fixes / features).

**Description:**
This PR configures SonarCloud in the repo, so that we can keep track of code quality in the future.

**Related issues:**
N/A

### CODE REVIEW

File: plugins/anthropic/src/claude.ts
Comment (EPMatt) [inline plugins/anthropic/src/claude.ts:1]: Not related to SonarCloud configuration, can we put this in a separate PR?

**Review by EPMatt — CHANGES_REQUESTED** [review]
Thanks @Dabolus, all good to me. I'm just requesting to remove a minor change not related to SonarCloud.

File: plugins/anthropic/src/claude.ts
Comment (Dabolus) [reply plugins/anthropic/src/claude.ts:1]: @EPMatt done

**Review by EPMatt — APPROVED** [review]
